### PR TITLE
Added fromText to ColorRgba

### PR DESCRIPTION
**Pull request by marcusnaslund**
Useful in the other project.
Peer review @davidpiuva ?

### CODE REVIEW

File: source/draw/Color.ooc [hunk 1]
```diff
@@ -153,6 +153,33 @@ ColorRgba: cover {
 	distance: func (other: This) -> Float { (this toRgb() distance(other toRgb()) * 3.0f + (this a - other a) as Float pow(2)) / 4.0f sqrt() }
 	operator == (other: This) -> Bool { this equals(other) }
 	operator != (other: This) -> Bool { !this equals(other) }
+
+	fromText: static func (text: Text) -> This {
```
Comment (davidpiuva) [inline source/draw/Color.ooc]: I guess that you just moved this function but if you use Text, you have to free it when given ownership.

Comment (marcusnaslund) [inline source/draw/Color.ooc]: Yes, I just moved it. The caller will free `text`.

File: test/draw/ColorConvertTest.ooc [hunk 2]
```diff
@@ -44,6 +44,10 @@ ColorConvertTest: class extends Fixture {
 			expect(color toRgb() equals(ColorRgb new(78, 99, 118)), is true)
 			expect(color toRgba() equals(ColorRgba new(78, 99, 118, 255)), is true)
 		})
+		this add("RGBA from text", func {
+			color := ColorRgba fromText(t"12,34,56,78")
+			expect(color equals(ColorRgba new(12, 34, 56, 78)))
```
Comment (davidpiuva) [inline test/draw/ColorConvertTest.ooc]: Monochrome, Monochrome+Alpha and RGB needs to be tested too.

File: source/draw/Color.ooc (hunk 1 above)
Comment (davidpiuva) [inline source/draw/Color.ooc]: That was the convention for String but with Text we can't assume that.

Comment (marcusnaslund) [inline source/draw/Color.ooc]: True, there should be `text free(Owner Receiver)` there.

File: test/draw/ColorConvertTest.ooc (hunk 2 above)
Comment (marcusnaslund) [inline test/draw/ColorConvertTest.ooc]: Fair enough.

**Comment by marcusnaslund** [comment]
Updated.

**Comment by davidpiuva** [comment]
Looks good

**Pull request merged**